src/story.ts

```diff
@@ -126,7 +131,7 @@ export type PlayFunction<TFramework extends AnyFramework = AnyFramework, TArgs =
 
 // This is the type of story function passed to a decorator -- does not rely on being passed any context
 export type PartialStoryFn<TFramework extends AnyFramework = AnyFramework, TArgs = Args> = (
-  update?: StoryContextUpdate<TArgs>
+  update?: StoryContextUpdate<Partial<TArgs>>
```

Comment:
Was this the change that allowed us to make decorators take `TArgs` again?

Reply:
Yes!